Library/Homebrew/formula.rb

```diff
@@ -459,6 +459,12 @@ def bottle_for_tag(tag = nil)
   # @see .version
   delegate version: :active_spec
 
+  delegate [ # rubocop:disable Layout/HashAlignment
```

Comment:
What's this look like without the disable? I think it's worth not disabling and just doing whatever that form looks like. Alternatively, if it really has to be disabled, please add a Ruby comment explaining why.

Reply:
If I run `brew --style fix` without disable:

```diff
--- a/Library/Homebrew/formula.rb
+++ b/Library/Homebrew/formula.rb
@@ -459,11 +459,11 @@ class Formula
   # @see .version
   delegate version: :active_spec

-  delegate [ # rubocop:disable Layout/HashAlignment
+  delegate [
     :allow_network_access,
     :deny_network_access,
     :network_access_allowed?,
-  ] => :"self.class"
+  ]                                                                                                 => :"self.class"

   # Whether this formula was loaded using the formulae.brew.sh API
   # @!method loaded_from_api?
   ```
   
I suspect this is why the check is also disabled already for an earlier spot (line 403). Not sure how to describe this other than "Stop RuboCop from erroneously indenting hash target" or something like that.